src/analyses/loopTermination.ml

```diff
@@ -52,7 +47,7 @@ struct
         (try
            let loop_statement = find_loop ~loop_counter in
            let is_bounded = check_bounded ctx loop_counter in
-           ctx.sideg () (G.add (`Lifted loop_statement) is_bounded (ctx.global ()));
+           ctx.sideg () is_bounded;
```

Comment:
Why is the more precise per-loop information removed now?

Reply:
Because it already exists in the local states where the warnings are emitted per-loop.
The `MustTermLoop` query is completely unused and just extra maintenance burden. Also, the used `MustTermAllLoops` query becomes more efficient because it becomes just a constant-time lookup.

Reply:
Ok, as the side-effect only happens only in postsolving there also isn't any precision advantage to keeping it per loop (which I initially thought we had there).